pkg/scheduler/framework/plugins/nodeports/node_ports.go

```
@@ -86,9 +87,9 @@ func (pl *NodePorts) PreFilterExtensions() framework.PreFilterExtensions {
 func getPreFilterState(cycleState *framework.CycleState) (preFilterState, error) {
 	c, err := cycleState.Read(preFilterStateKey)
 	if err != nil {
-		// preFilterState state doesn't exist. We ignore the error for now since
-		// Filter is able to handle that by computing it again.
-		return nil, nil
+		// preFilterState state doesn't exist.
+		klog.V(5).Infof("Error reading %q from cycleState: %v", preFilterStateKey, err)
+		return nil, err
```

Ditto. We can simply return a verbose err and remove the logging.